codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -148,28 +152,26 @@ var Worksheet = React.createClass({
           $("#raw-textarea").focus();
         }
     },
-    toggleActionBar: function(){
-        this.setState({showActionBar:!this.state.showActionBar});
+    toggleActionBar: function() {
+        this.setState({showActionBar: !this.state.showActionBar});
     },
-    showActionBar: function(){
-        this.setState({showActionBar:true});
+    showActionBar: function() {
+        this.setState({showActionBar: true});
     },
-    hideActionBar: function(){
-        this.setState({showActionBar:false});
+    hideActionBar: function() {
+        this.setState({showActionBar: false});
     },
-    refreshWorksheet: function(){
+    refreshWorksheet: function() {
         $('#update_progress').show();
-        this.setState({updating:true});
+        this.setState({updating: true});
         ws_obj.fetch({
-            success: function(data){
-                // console.log("fetch_and_update success");
-                // console.log("%c--------------------------------------------------------------", "color: Green; font-size:15px;");
-                if(this.isMounted()){
+            success: function(data) {
+                if (this.isMounted()) {
                     this.refs.list.setState({worksheet: ws_obj.getState()});
```

Comment:
I'm not able to reproduce this...do you have the latest CLI, and can you be
more specific about what you tried and what error you got?

On Sun, Oct 18, 2015 at 11:07 PM, Pujun Bhatnagar <notifications@github.com>
wrote:

> In codalab/apps/web/static/js/worksheet/worksheet_interface.js
> <https://github.com/codalab/codalab/pull/1342#discussion_r42338422>:
>
> >          ws_obj.fetch({
> > -            success: function(data){
> > -                // console.log("fetch_and_update success");
> > -                // console.log("%c--------------------------------------------------------------", "color: Green; font-size:15px;");
> > -                if(this.isMounted()){
> > +            success: function(data) {
> > +                if (this.isMounted()) {
> >                      this.refs.list.setState({worksheet: ws_obj.getState()});
>
> Also, the AJAX call in line 231 of worksheet_sidepanel sees to be giving
> error 500 (internal server error) when user clicks on a bundle that is a
> dataset and does not have any associated sub-files with it. Here is a
> screenshot:
> [image: error]
> <https://cloud.githubusercontent.com/assets/5567951/10570813/eb2d84aa-75ec-11e5-94eb-fb6833a51eb3.png>
>
> Should we handle this at the cli side?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab/pull/1342/files#r42338422>.
>